RobotCode/src/main/java/frc/robot/commands/RotateShooterMountToPositionCommand.java

```diff
@@ -1,60 +1,36 @@
 package frc.robot.commands;
 
-import java.util.function.DoubleSupplier;
-
 import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ShooterMountConstants;
 import frc.robot.subsystems.ShooterMountSubsystem;
 
 public class RotateShooterMountToPositionCommand extends Command
 {
 
-
-    private final ShooterMountSubsystem ShooterMountSubsystem;
-
-    private DoubleSupplier getPosition;
-
-    private boolean endAutomatically;
-
-    /**
-     * This constructor is the version that allows for variable targets (ex: based on joystick)
-     * 
-     * @param getPosition in degrees
-     */
-    public RotateShooterMountToPositionCommand(ShooterMountSubsystem subsystem,
-            DoubleSupplier getPosition, boolean endAutomatically)
-    {
-        ShooterMountSubsystem = subsystem;
-        this.getPosition = getPosition;
-        this.endAutomatically = endAutomatically;
-
-        // We do this before setting the rotation so that any other
-        // command controlling the shooter mount will end before we set the rotation
-        addRequirements(subsystem);
-    }
+    private ShooterMountSubsystem shooterMount;
+	  private double targetPosition;
 
     /**
      * This constructor is the version that uses a constant target
      * 
      * @param position in degrees
      */
-    public RotateShooterMountToPositionCommand(ShooterMountSubsystem subsystem, double position,
-            boolean endAutomatically)
+    public RotateShooterMountToPositionCommand(ShooterMountSubsystem shooterMount, double targetPosition)
     {
-        this(subsystem, () -> position, endAutomatically);
-    }
+        this.shooterMount = shooterMount;
+		    this.targetPosition = targetPosition;
 
-    @Override
-    public void execute()
-    {
-        ShooterMountSubsystem.setGoalRotation(getPosition.getAsDouble());
+		addRequirements(shooterMount);
     }
 
     @Override
-    public boolean isFinished()
+    public void initialize()
     {
-        // We do the weird static access to avoid going through the variable
-        return endAutomatically && Math.abs(ShooterMountSubsystem.getCurrentRotation()
-                - getPosition.getAsDouble()) < frc.robot.subsystems.ShooterMountSubsystem.DEADBAND;
+        shooterMount.setTargetRotation(targetPosition);
     }
 
+	  public void end() {
+		    shooterMount.setTargetRotation(ShooterMountConstants.SHOOTER_MOUNT_MIN_ANGLE);
```

Comment:
Any particular reason we're going back down instead of holding position? If there is, add a javadoc to the class mentioning that it will go back down when the command ends.